autoscheduler/frontend/src/tsconfig.json

```diff
@@ -5,6 +5,10 @@
         "noImplicitAny": true,
         "module": "commonjs",
         "target": "es2018",
+        "typeRoots": [
+            "./node_modules/@types",
+            "./globals.ts",
+        ],
```

Comment:
Does this remove the need of having to export `STATIC_URL` and `IS_PRODUCTION`?

Reply:
Yes, declaring them in `globals.ts` makes typescript recognize them and putting them in `globals` in the eslintrc will make the linter recognize them